### DISCUSSION TITLE
添加youutbe视频朗读功能

### DISCUSSION THREAD

**Discussion by copdips** [immersive-translate/immersive-translate#1366] [Ideas]
### 新功能 | New Feature

youtube字幕可以翻译了, 能否开发一个覆盖原视频声音, 然后朗读翻译的字幕
类似于这个插件:
https://chromewebstore.google.com/detail/youtube%E4%B8%AD%E6%96%87%E9%85%8D%E9%9F%B3/oglffgiaiekgeicdgkdlnlkhliajdlja
这个插件有个小问题就是他使用的youtube自带的字幕翻译, 经常翻译的不好, 还是沉浸式的翻译加上chatgpt翻译的完美

### 为什么您想要添加这些功能 | Why you want to add these features

看视频希望听到中文更舒服, 谢谢!

**Comment by theowenyoung** [comment]
很好的idea～

但是这个功能加上的话，可能会让插件变得太臃肿，所以暂时应该没有计划。

**Reply by copdips** [reply]
谢谢回复, 期待有一天可以添加, 那时这个插件就nb大了, 这将是多少人的福音啊 :)
